Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
Ah, I see. I guess we could stick in a symlink from ` var/postgres-9.4 ` to ` var/postgres ` for a period of time? As long as we retain the ` plist ` changes, we should (*probably slowly*) get people off of using the unversioned ` var ` dir.

> PG_VERSION

This makes sense. How do you want to handle reading the file? I thought we could probably open it with ` File.open(var/"postgres/PG_VERSION").read ` but I wasn't sure how to then throw the contents at Brew in a way that wouldn't break everything, heh.